treeherder/intermittents_commenter/tasks.py

```diff
@@ -0,0 +1,12 @@
+from celery import task
+
+from treeherder.intermittents_commenter.commenter import Commenter
+
+
+@task(name='intermittents-commenter')
+def run_commenter(**kwargs):
+    """
+    Run the intermittents commenter in either daily or weekly mode.
+    """
+    process = Commenter(weekly_mode=False, dry_run=False)
```

Comment:
The options passed to the task aren't currently passed to `Commenter`. This needs to be:
```python
process = Commenter(weekly_mode=weekly_mode)
```